include/swift/Option/Options.td

```diff
@@ -1805,6 +1805,16 @@ def nostartfiles:
          HelpHidden, NewDriverOnlyOption]>,
   HelpText<"Do not link in the Swift language startup routines">;
 
+def enable_cas: Flag<["-"], "enable-cas">,
```

Comment:
Does this enable *caching* or just *cas*? In clang we have
* -fcache-compile-job
* -fcas-fs / -fcas-include-tree

and I'm wondering if this swift option should be "-cache-compile-job"?

Reply:
I currently think two options:

1) Rename as `-experimental-cas` and we can always rename it.
2) Make this a `NoDriverOption` and add a driver option `-cache-compile-job` later for enable caching specifically which implies CAS for depscan and compile

Reply:
I prefer (2) since it matches clang, but either of these is fine with me.